packages/cfd/src/Helpers/compare-accounts-config.ts

```diff
@@ -1,6 +1,8 @@
-import { TTradingPlatformAvailableAccount, TIconData } from '../Components/props.types';
+import { TIconData, TAvailableCFDAccounts, TModifiedTradingPlatformAvailableAccount } from '../Components/props.types';
 
-const getHighlightedIconLabel = (trading_platforms: TTradingPlatformAvailableAccount): TIconData[] => {
+type TCFDConfig = string;
```

Comment:
There's no need to have a custom type when you just use string as its type which can be added directly without defining a custom type. I suggest you add literal types for your CFDConfig.

Reply:
sure